openoa/utils/plot.py

```diff
@@ -1287,6 +1287,7 @@ def plot_distributions(
     figure_kwargs: dict = {},
     plot_kwargs: dict = {},
     annotate_kwargs: dict = {},
+    title: str=None,
```

Comment:
Should be `title: str | None = None`. I know we're not technically doing any static type checking, but could you update this?

Reply:
I made this change.